src/assets/styles/styles/custom/_button.scss

```diff
@@ -0,0 +1,33 @@
+.btn {
+  background: none;
+  border: none;
+  padding: 5px 20px 5px;
+  border-radius: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial,
+    sans-serif;
+  font-size: 15px;
+  font-weight: medium;
+  text-align: center;
+  transition: background-color 0.5s, color 0.5s;
+  &:hover {
+    cursor: pointer;
```

Comment:
Po wybraniu przycisku pojawia się wokól niego ramka. Wydaje mi się, że nie powinno jej być. Nie jest to jakoś bardzo istotne, ale jak znajdziesz chiwlkę to spróbuj się jej pozbyć :)